.github/workflows/ci.yml

```diff
@@ -10,12 +10,11 @@ jobs:
     strategy:
       matrix:
         include:
-          - java: 8
           - java: 11
```

Comment:
Should we add `java: 17` here as well?

Reply:
no unless we add "unused" methods to all tests - I hope we eventually will have a better fix :) one option is to create a new separate app, use parboild as dependency and run one test in jdk17....